src/group.h

```diff
@@ -24,13 +24,14 @@ extern GroupPool _group_pool; ///< Pool of groups.
 /** Statistics and caches on the vehicles in a group. */
 struct GroupStatistics {
 	uint16 num_vehicle;                     ///< Number of vehicles.
+	Money profit_last_year;                 ///< Sum of profits for all vehicles.
 	uint16 *num_engines;                    ///< Caches the number of engines of each type the company owns.
 
 	bool autoreplace_defined;               ///< Are any autoreplace rules set?
 	bool autoreplace_finished;              ///< Have all autoreplacement finished?
 
 	uint16 num_profit_vehicle;              ///< Number of vehicles considered for profit statistics;
-	Money profit_last_year;                 ///< Sum of profits for all vehicles.
+	Money profit_last_year_min_age;         ///< Sum of profits for considered vehicles.
```

Comment:
Naming of these variables does need to be improved/unified in some manner.
There is `num_vehicle` and `num_profit_vehicle` vs `profit_last_year` and `profit_last_year_min_age`. This brings a whole load of questions to me, as `num_profit_vehicles` is not linked to `profit_last_year` though the comment somewhat implies that.
So, some sort of unified naming should be devised so it becomes easier to link the two sets of variables together. I'm not quite sure what to use though. However, I do not really like `min_age` to be used as that might not be all (now or in the future). In other words, it should be about the concept of considering it for performance ratings and not the way the consideration has been implemented.

Reply:
I tried to come up with a better naming. `num_vehicle`/`profit_last_year` are related to each other, and `num_vehicle_min_age`/`profit_last_year_min_age` is the other duo.
```
/** Statistics and caches on the vehicles in a group. */
struct GroupStatistics {
	uint16 num_vehicle;                     ///< Number of vehicles.
	Money profit_last_year;                 ///< Sum of profits for all vehicles.
	uint16 *num_engines;                    ///< Caches the number of engines of each type the company owns.

	bool autoreplace_defined;               ///< Are any autoreplace rules set?
	bool autoreplace_finished;              ///< Have all autoreplacement finished?

	uint16 num_vehicle_min_age;             ///< Number of vehicles considered for profit statistics;
	Money profit_last_year_min_age;         ///< Sum of profits for considered vehicles.
```